src/entities/profile/model/types.ts

```diff
@@ -3,14 +3,35 @@
  * Use branded type to entity id to
  * don't to be confused with other identifiers
  */
-export type ProfileId = Brand<Id, 'ProfileId'>
+interface IUserFavorite {
+  id: number,
+  name: string
+}
+// export type ProfileId = Brand<Id, 'ProfileId'>
+export type ProfileId = {userId: string};
```

Comment:
Use an interface instead of a type.eslint

https://habr.com/ru/sandbox/186102/ - types vs interfaces